draft-nro-bulk-rdap.md

```diff
@@ -19,21 +19,21 @@ email = "secretariat@nro.net"
 
 .# Abstract
 
-To complement the move from Whois to RDAP for the RIRs (Regional Internet Registries), this document specifies a new
-service named the Bulk RDAP that an RIR can deploy to replace their Bulk Whois service.
+To complement the move from Whois to RDAP for the Regional Internet Registries (RIRs), this document specifies a new
+service, named the Bulk RDAP, that an RIR can deploy in lieu of its Bulk Whois service.
 
 {mainmatter}
 
 # Introduction
 
-As the RIRs shift from Whois to RDAP, they also need an RDAP replacement for their Bulk Whois service. To that end, this
-document specifies a new service named the Bulk RDAP that an RIR can deploy in lieu of the Bulk Whois. This service is
-intended to be a simple, easy-to-implement replacement for the Bulk Whois.
+As the RIRs shift from Whois to RDAP, they also need an RDAP counterpart for their Bulk Whois service. To that end, this
+document specifies a new service, named the Bulk RDAP, that an RIR can deploy to replace the Bulk Whois. This service is
```

Comment:
to replace the bulk Whois -> to provide access to RDAP data in bulk format

Reply:
Agreed.